bin/mail/2016-05-02-petition-followup/send-test-mails

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+../send-to-mailing-list 5-2-petition-follow-up ../mailing-common/test-params-us.csv
```

Comment:
Yes, and that it's formatted correctly. I always like to send myself all variations of the emails first to check formatting before sending them out to mass audience.